### PR TITLE
(8.5.4) GH-2166: Update name of Unknown category

### PR DESCRIPTION
**Pull request by benstrumeyer**
* [x] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

- Update name of unknown category

Ticket: https://ghostery.atlassian.net/browse/GH-2166

### CODE REVIEW

**Review by wlycdgr — COMMENTED** [review]
Same deal here as with GH-2178: let's update the variable names in the code to try to keep everything consistent. There are a lot of places where we use `unknown` but the large majority of them look like they should be a straightforward search&replace situation.

Bigger picture, we should think about whether there might be a safer and more efficient way we can keep this stuff in sync. It would def be better if we could make these kind of copy updates without needing to manually update a bunch of code like this.

**Review by wlycdgr — COMMENTED** [review]
The panel's `unkown.svg` file also needs to be renamed - right now the icon is broken cos it's looking for `unidentified.svg`

File: CHANGELOG.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ See the complete GitHub [milestone](https://github.com/ghostery/ghostery-extensi
 + Migrate to the new Chromium-based Edge browser (#492)
 + Disable Purplebox on Firefox Android (#494)
 + Allow white-listing of wildcard domains (#501)
-+ Allow site-specific white-listing of Unknown Trackers caught by Ad-Blocker (#503)
++ Allow site-specific white-listing of Unidentified Trackers caught by Ad-Blocker (#503)
```
[the remark this answers is not in the document]
Comment (benstrumeyer) [reply CHANGELOG.md]: That makes sense, left those as is 👍

**Review by wlycdgr — APPROVED** [review]
LGTM

**Pull request merged**